### PR TITLE
Update IoTA-Sigfox with the changes of the IoTA Node Lib related to multi-threading

### PR DESCRIPTION
**Pull request by flopezag**
#### REVIEWER
@fgalan

#### DESCRIPTION
- Modify the Sigfox IoT Agent to deal with the option multi-thread.
- Add the corresponding new variable in the config.js (multiCore) with description.
- Change in the Readme.md (Docker) and Administration Guide regarding High Performance execution of the Sigfox IoT Agent.
- Create 2 examples files of the user limits and sysctl configuration for high performance execution.
- Correct typos in doc files.
- Provide information of the Sigfox IoT Agent version in the request /iot/about.
- Improve the testing script (sigfox-test.js)
- Improve the information about mandatory fields not provided (errors.js & sigfoxHandlers.js).
- (Temporally) Update the package.json to link to the IoTAgent Node Lib with the changes for testing purposes. NEED TO BE REVERT when the PR #815 is merged.

#### TESTING
Unit tests, E2E tests in single-thread (legacy options) and multi-thread:

- Obtain IoT Agent JSON version info
- Provisioning Service
- Retrieve Service
- Provisioning Device
- Retrieve all devices
- Retrieve 1 device
- (using sigfox-test) set parameters
- (using sigfox-test) show parameters
- (using sigfox-test) send measure

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -135,34 +135,35 @@ config.iota = {
...
-     * coordinated by an IoT Manager, this section defines the information that will be used to connect with that manager.
+     * coordinated by an IoT Manager, this section defines the information that will be used to connect with that
+     * manager.
      */
-    iotManager: {
-        /**
-         * Host where the IoT Manager is located.
-         */
-        host: 'localhost',
+    //iotManager: {
```
Comment (fgalan) [inline config.js]: I understand the lines commented with `//` were some test but they should be changed back.

File: package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "command-node": "0.1.1",
     "request": "2.88.0",
     "async": "1.5.2",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/flopezag/iotagent-node-lib.git#feature/cluster-support",
```
Comment (fgalan) [inline package.json]: Just a reminder: change this back

File: config.js (hunk 1 above)
Comment (jason-fox) [reply config.js]: Fixed: 333f0ec

File: package.json (hunk 2 above)
Comment (jason-fox) [reply package.json]: Fixed: 333f0ec

**Pull request merged**